tools/packaging/kata-monitor/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM golang:1.15-alpine
+
+RUN apk add bash curl git make
+WORKDIR /go/src/github.com/kata-containers/kata-containers/src/runtime
+COPY . /go/src/github.com/kata-containers/kata-containers
+RUN SKIP_GO_VERSION_CHECK=true make monitor
+
+FROM alpine:latest
```

Comment:
If minimal container is a requirement, what about `FROM scratch` instead?

Reply:
I find it useful to still be able to do basic debug / view inside the container. There's only one monitor per node, so it isn't a major issue. I would rather use alpine over something like ubuntu, though!